### PR TITLE
EAR 1770 Add BICS changes

### PR DESCRIPTION
**Pull request by farres1**
### What is the context of this PR?

Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review

Describe the steps required to test the changes (include screenshots if appropriate).

### CODE REVIEW

**Review by jasehumphr — APPROVED** [review]

**Review by jchapman68 — APPROVED** [review]

File: schemas/business/en/covid_0001.json [hunk 1]
```diff
@@ -526,78 +534,64 @@
                     "id": "answer081cb8c7-b8b1-465f-a237-808cb9bd93c5",
                     "mandatory": false,
                     "type": "Checkbox",
-                    "label": "Select all that apply:",
                     "options": [
                       {
                         "label": "Additional paperwork",
                         "value": "Additional paperwork",
-                        "q_code": "681",
```
[the remark this answers is not in the document]
Comment (farres1) [reply schemas/business/en/covid_0001.json:534]: I was unsure about this too. The QCodes had been removed from the questionnaire in Author.

[the remark this answers is not in the document]
Comment (farres1) [reply schemas/business/en/covid_0001.json:534]: From the Author team's product manager:
> The business said the dummy codes had been added. So I'd assume yes. Deliberate

[the remark this answers is not in the document]
Comment (berroar) [reply schemas/business/en/covid_0001.json:534]: I think they would get an error on submission without these q_codes (or for any selected answer where q_codes are missing) so they probably need to be added

**Review by ashtuck — APPROVED** [review]

**Review by martyncolmer — APPROVED** [review]

**Review by oms09 — APPROVED** [review]

**Review by jasehumphr — APPROVED** [review]

File: schemas/business/en/covid_0001.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MebinAbraham) [reply schemas/business/en/covid_0001.json:534]: Resolving. Looks good.

**Pull request merged**